utils/validateUser.js

```diff
@@ -0,0 +1,49 @@
+module.exports.validateRegisterInput = (
+  firstName,
+  lastName,
+  email,
+  password,
+  confirmPassword,
+) => {
+  const errors = {};
+
+  if (firstName.trim() === '') {
```

Comment:
Please, use `joi-validator` for the validation

Reply:
fixed